openwisp_monitoring/check/tasks.py

```diff
@@ -16,7 +18,7 @@ def get_check_model():
     return load_model('check', 'Check')
 
 
-@shared_task
+@shared_task(time_limit=60 * 60)
```

Comment:
please increase to 2 hours just in case we hit this limit, in the future we'll probably have to make this configurable for very large instances... unless this 2 hours limit never gets hit, but if it does it will be a great problem to have! :grin: but we can deal with it later so for now just increase it to 2 hours.